httpstan/views.py

```diff
@@ -273,7 +273,10 @@ async def handle_show_params(request: aiohttp.web.Request) -> aiohttp.web.Respon
     for name, dims_ in zip(param_names, dims):
         constrained_names = tuple(filter(lambda s: re.match(fr"^{name}\.\S+|^{name}\Z", s), constrained_param_names))
         assert isinstance(dims_, list)
-        assert constrained_names, constrained_names
+        if 0 in dims_:
+            assert not constrained_names, constrained_names
+        else:
+            assert constrained_names, constrained_names
```

Comment:
```suggestion
```

Let's just get rid of the assertion.

But we do need a test which has vector parameter of length zero. Probably could use one that has both a zero-length parameter and a non-zero length parameter.